src/core/hiwire.c

```diff
@@ -118,6 +118,11 @@ EM_JS_NUM(errcode, hiwire_decref, (JsRef idval), {
   Module.hiwire.decref(idval);
 });
 
+EM_JS_NUM(errcode, hiwire_log_error, (char* msg), {
```

Comment:
Does this belong to the hiwire namespace?

Reply:
I don't know. What is the purview of the hiwire namespace? This is a short `EM_JS` function. I suppose your reasoning must be that it takes a `char* msg` and takes care of alloc and dealloc whereas hiwire mostly would take a `JsRef` and not explicitly allocate? It would be a better api if it took a `JsRef` though.

I could see moving it to `error_handling.c` in either case. `hiwire` is a bit of a weird file anyways. Most of the stuff in there is helpers for `JsRef` with helpers for random other stuff mixed in.

Reply:
error_handling would make more sense to me.

Reply:
Okay, will do that then. Thanks Dexter!